website/templates/report.html

```diff
@@ -367,6 +381,24 @@ <h2 class="text-2xl font-semibold leading-7 text-gray-900">{% trans "ADD TEAM ME
 {% endblock content %}
 {% block after_js %}
     <script type="text/javascript">
+    function isCVEValid(cveNumber) {
+        const cveRegex = /^CVE-\d{4}-\d{4,}$/;
+        return cveRegex.test(cveNumber);
+    }
+
+    function validateCVE() {
+        const cveInput = document.getElementById('cve_no_input').value;
+        const validationResult = document.getElementById('cve_no_input');
+
+        if (isCVEValid(cveInput)) {
+            validationResult.style.borderColor = 'green';
+        }  else if (cveInput == None){
```

Comment:
What's `None` here?

Reply:
This is JS context, not Python. It does not have `None`, right?

Reply:
I forget to remove it but have hade these changes in #1974